react/src/objects/stack/SprkStack.test.js

```diff
@@ -9,7 +9,7 @@ Enzyme.configure({ adapter: new Adapter() });
 describe('SprkStack:', () => {
   it('should display a div element with the correct base class', () => {
     const wrapper = shallow(<SprkStack />);
-    expect(wrapper.find('div.sprk-o-Stack').length).toBe(1);
+    expect(wrapper.html()).toEqual('<div class="sprk-o-Stack" data-id=""></div>');
```

Comment:
Informational:
We `data-id=''` as a default so it always renders the `data-id` whether it is being passed or not. So I had to include it here.